app/services/irs_attempts_api/event.rb

```diff
@@ -0,0 +1,65 @@
+module IrsAttemptsApi
+  Event = Struct.new(:jti, :iat, :event_type, :encrypted_event_data, keyword_init: true) do
+
+    def self.build(event_type:, session_id:, occurred_at:, event_metadata:)
+      jti = SecureRandom.uuid()
+      iat = Time.now.to_i
+      encrypted_event_data = encrypt_event_data(
+        session_id: session_id,
+        occurred_at: occurred_at,
+        event_metadata: event_metadata,
+      )
+      self.new(
+        jti: jti, iat: iat, event_type: event_type, encrypted_event_data: encrypted_event_data
+      )
+    end
```

Comment:
one thought... why bother building a JWT and then encrypting? What if we just JWE encrypted the JSON payload? The VA Inherited Proofing API uses encrypted JWEs: https://github.com/18F/identity-idp/pull/6307/files#diff-66368066c912fa2ac741918f22a51221332ebbc6b7b6c04308e6bdec701d6dbbR27

Reply:
It'd take a little rewiring. We need to keep track of at least the JTI in plaintext so we can go back around and delete events after they ACK them.

May also take some back and forth with IRS, but let me take a swing.

Reply:
Okay, just pushed a bunch of stuff to switch to JWE. I got rid of `Event` since we can no longer actually read events from Redis since they are all JWEed now.